examples/dem/3d-rotating-drum/post_processing_rotating_drum.py

```diff
@@ -0,0 +1,109 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 The Lethe Authors
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception OR LGPL-2.1-or-later
+
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import argparse
+from lethe_pyvista_tools import *
+from numpy.ma.core import arctan
+from numpy.ma.extras import average
+
+parser = argparse.ArgumentParser(description='Arguments for calculation of the velocity profile in the rotating drum')
+parser.add_argument("-f", "--folder", type=str, help="Folder path", required=True)
+parser.add_argument("-i", "--input", type=str, help="Name of the input file", required=True)
```

Comment:
What is the input file here? is it the prm file? 
If it's the prm file, it woud be better to use --prm for the prm file instead it would be more explicit

Reply:
Is is not the prm, it is the csv file (experimental data).

I can add an other flag for the prm. 

The prm is loaded explicitly at line 31.